pandas/tests/computation/test_eval.py

```diff
@@ -1345,6 +1345,38 @@ def test_multi_line_expression_local_variable(self):
         assert_frame_equal(expected, df)
         assert ans is None
 
+    def test_multi_line_expression_callable_local_variable(self):
```

Comment:
so this works currently yes?

Reply:
Prior to this fix only positional arguments were working.

Reply:
> add a whatsnew note; reshaping or other section would be ok

I've added an entry in the _other_ section